forward/request_sender.go

```diff
@@ -91,8 +91,10 @@ func (s *requestSender) Send() (res []byte, err error) {
 			return s.ScheduleRetry()
 		}
 
+		identity, _ := s.sender.WhoAmI()
```

Comment:
how likely is it that there will be an error here? I can imagine that Send already errored earlier if ringpop is not initialized. Otherwise we should take the error here and handle it.